websites/documentation/gatsby-config.js

```diff
@@ -20,5 +20,12 @@ module.exports = {
       additionalPrismLanguages: ['java', 'scala', 'csharp', 'swift', 'php'],
       addOns: ['@commercetools-docs/gatsby-theme-code-examples'],
     }),
+    {
+      resolve: '@commercetools-docs/gatsby-theme-sso-ui-kit',
+      options: {
+        auth0Domain: 'commercetools-professionals.eu.auth0.com',
+        auth0ClientId: 'nZwQWPaoTq8IIk67VV8o0Ska9lGp4hnW',
```

Comment:
Is that ok to hardcode them?

Reply:
This is just for testing...will be removed as soon as the env variables work as expected.